<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgAnimationSelector</class>
 <widget class="QWidget" name="WdgAnimationSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="lblAnimationName">
        <property name="text">
         <string>Animation name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLocation">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAuthor">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDescription">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLineEdit" name="txtAnimationName"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="txtLocation"/>
        </item>
        <item>
         <widget class="QPushButton" name="locationButton">
          <property name="text">
           <string>Change...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="txtAuthor"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDescription"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>611</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Animation Size</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>441</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="lblWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblHeight">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblResolution">
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QDoubleSpinBox" name="inputWidth">
         <property name="maximum">
          <double>1000000.989999999990687</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="inputHeight">
         <property name="maximum">
          <double>1000000.989999999990687</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="inputResolution">
         <property name="maximum">
          <double>100000.990000000005239</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QComboBox" name="inputWidthUnit"/>
       </item>
       <item>
        <widget class="QComboBox" name="inputHeightUnit"/>
       </item>
       <item>
        <widget class="QLabel" name="lblPpi">
         <property name="text">
          <string>ppi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>611</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Animation Options</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="lblFps">
         <property name="text">
          <string>FPS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblTime">
         <property name="text">
          <string>Duration(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblOpacity">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblBackground">
         <property name="text">
          <string>Background:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QSpinBox" name="inputFps"/>
       </item>
       <item>
        <widget class="QSpinBox" name="inputTime"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="lblTransparent">
           <property name="text">
            <string>Transparent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="inputOpacity">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOpaque">
           <property name="text">
            <string>Opaque</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KColorButton" name="inputBackground"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>611</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Color</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KisColorSpaceSelector" name="colorSpaceSelector" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="bnCreateAnimation">
      <property name="text">
       <string>Create Animation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>519</y>
     <width>611</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Open Existing Animation</string>
   </property>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Animation</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblFile">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtOpenFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectFile">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KisColorSpaceSelector</class>
   <extends>QWidget</extends>
   <header>widgets/kis_color_space_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
